pages/linux/multipass.md

```diff
@@ -0,0 +1,36 @@
+# multipass
+
+> Creates a Ubuntu virtual machine using native hypervisors.
+> More information: <https://multipass.run/>.
+
+- List the aliases that can be used to launch an instance:
+
+`multipass find`
+
+- Launch a new instance, set its name, use a cloud-init configuration file:
```

Comment:
```suggestion
- Launch a new instance and set its name using a cloud-init configuration file:
```

Reply:
- The 3 files are similar but there's none for sunos. So should it go to common anyway?
- "using" is not appropriate: the name is set using the -n flag while the config file (which does not define the name) is set using --cloud-init.

Reply:
- Yep. `common/` is for 2 or more platforms, especially for 3. :smile:
-  Then I probably didn't understand the text here. Can you try to rephrase what you meant more clearly?

Reply:
- Moved to common :)
- I don't really know how to make it clearer... Here's from the official [doc](https://multipass.run/docs/launch-command):
  - -n, --name <name>    Name for the instance.
  - --cloud-init <file>  Path to a user-data cloud-init configuration

I can remove that --cloud-init flag, but that would be a pity since it's a pretty useful one

Reply:
If I understand correctly they are just unrelated options, so:

> Launch a new instance, set its name, _and_ use a cloud-init configuration file:

Reply:
You understand correctly :)
Applied as-is, only removed the comma before the "and".